app/models/subnet/ipv4.rb

```diff
@@ -0,0 +1,42 @@
+require 'socket'
+
+class Subnet::Ipv4 < Subnet
+  def self.default_sti_class
+    'Subnet::Ipv4'
```

Comment:
is this really required? I would assume its only needed on the top level parent class?